ownersort.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+find -user $USER|while read i
+do
+  cp $i /home/$USER/$i
```

Comment:
^-- SC2086: Double quote to prevent globbing and word splitting.
              ^---^ SC2086: Double quote to prevent globbing and word splitting.
                    ^-- SC2086: Double quote to prevent globbing and word splitting.